app/lab/package.json

```diff
@@ -200,6 +200,7 @@
       "@jupyterlite/server-extension"
     ],
     "singletonPackages": [
+      "@jupyter-notebook/ydoc",
```

Comment:
TODO: this will be updated to `@jupyter/ydoc`: https://github.com/jupyterlab/jupyterlab/pull/13506